src/test/regress/expected/multi_utilities.out

```diff
@@ -317,12 +356,129 @@ WHERE tablename = 'dustbunnies_990002' ORDER BY attname;
 VACUUM;
 WARNING:  not propagating VACUUM command to worker nodes
 HINT:  Provide a specific table in order to VACUUM distributed tables.
+-- check the current number of vacuum and analyze run on dustbunnies
+SELECT run_command_on_workers($$SELECT wait_for_stats()$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,"")
+ (localhost,57638,t,"")
+(2 rows)
+
+SELECT run_command_on_workers($$SELECT pg_stat_get_vacuum_count(tablename::regclass) from pg_tables where tablename LIKE 'dustbunnies_%' limit 1$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,3)
+ (localhost,57638,t,3)
+(2 rows)
+
+SELECT run_command_on_workers($$SELECT pg_stat_get_analyze_count(tablename::regclass) from pg_tables where tablename LIKE 'dustbunnies_%' limit 1$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,3)
+ (localhost,57638,t,3)
+(2 rows)
+
 -- and warning when using targeted VACUUM without DDL propagation
 SET citus.enable_ddl_propagation to false;
 VACUUM dustbunnies;
 WARNING:  not propagating VACUUM command to worker nodes
 HINT:  Set citus.enable_ddl_propagation to true in order to send targeted VACUUM commands to worker nodes.
+ANALYZE dustbunnies;
+WARNING:  not propagating ANALYZE command to worker nodes
+HINT:  Set citus.enable_ddl_propagation to true in order to send targeted ANALYZE commands to worker nodes.
 SET citus.enable_ddl_propagation to DEFAULT;
+-- should not propagate the vacuum and analyze
+SELECT run_command_on_workers($$SELECT wait_for_stats()$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,"")
+ (localhost,57638,t,"")
+(2 rows)
+
+SELECT run_command_on_workers($$SELECT pg_stat_get_vacuum_count(tablename::regclass) from pg_tables where tablename LIKE 'dustbunnies_%' limit 1$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,3)
+ (localhost,57638,t,3)
+(2 rows)
+
+SELECT run_command_on_workers($$SELECT pg_stat_get_analyze_count(tablename::regclass) from pg_tables where tablename LIKE 'dustbunnies_%' limit 1$$);
+ run_command_on_workers 
+------------------------
+ (localhost,57637,t,3)
+ (localhost,57638,t,3)
+(2 rows)
+
+-- should not create the index in workers
```

Comment:
Do we have _any_ evidence that index creation/removal, truncation, etc. are not propagated? We shouldn't just add tests without reason (otherwise these test files get bloated and impossible to follow).

Since this PR is about `VACUUM`/`ANALYZE`, can you remove the index/truncate tests?